CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.14.0] - 2021-03-11
+
+### Changed
+
+- Resolves #79 - Bump plonk to 0.6
```

Comment:
This should be:
```
- Update `plonk` from `0.5` to `v0.6` #79
```

"Bump" should be used only for the current crate (and it never goes on the changelog)

Reply:
Fixed